pom.xml

```diff
@@ -58,9 +58,9 @@
         <sortpom-maven-plugin.version>2.8.0</sortpom-maven-plugin.version>
         <spring-beans.version>5.3.21</spring-beans.version>
         <spring-core.version>5.3.21</spring-core.version>
-        <springfox.version>3.0.0</springfox.version>
+        <springdoc.version>1.5.10</springdoc.version>
```

Comment:
Please use the latest springbok version available. I see it as 1.6.12 as of today.